projects/arlas-components/src/lib/components/mapgl/model/ArlasMapboxGL.ts

```diff
@@ -353,46 +352,46 @@ export class ArlasMapGL extends AbstractArlasMapGL implements MapOverride {
           const layer = this.layersMap.get(l);
```

Comment:
This method could be moved to the abstract map, since we only use methods that are reimplemented by each map provider

Reply:
fixed